src/Features/CSharp/Portable/InitializeParameter/CSharpAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -37,6 +40,16 @@ protected override void InsertStatement(SyntaxEditor editor, SyntaxNode function
         protected override bool IsImplicitConversion(Compilation compilation, ITypeSymbol source, ITypeSymbol destination)
             => InitializeParameterHelpers.IsImplicitConversion(compilation, source, destination);
 
+        protected override IEnumerable<SyntaxNode> GetParameterList(SyntaxNode parameterNode)
```

Comment:
IEnumerable will allocate.  Can we not just return `SyntaxList<SyntaxNode>`?

Reply:
Changed it to ImmutableArray instead.